01_login.php

```diff
@@ -73,29 +95,46 @@
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-padding w3-card">
+				<div class="w3-left">
 				<img src="images/placeholder.jpg" width="40" height="40">
+				</div>
 				<a class="w3-bar-item"><b>Soundwave</b> </a>
 			</div>
 		</div>
+		
 
 
 		
 		<!-- background container; colour orange -->
-		<div class="container2" style="max-width:100%">
-		
+		<div class="w3-orange container w3-image">
+			<div class="bg">
```

Comment:
added new image within the background: calls upon the image from my internal styles sheet